src/linux_syscall_support.h

```diff
@@ -1,4 +1,4 @@
-/* Copyright (c) 2005-2008, Google Inc.
+/* Copyright (c) 2005-2011, Google Inc.
```

Comment:
I didn't review this, I hope it's right :)

Reply:
The version at 2011 is so perfect that no new versions are necessary. :-)